app/src/main/java/com/github/houseorganizer/houseorganizer/panels/MainActivity.java

```diff
@@ -26,11 +26,12 @@ protected void onCreate(Bundle savedInstanceState) {
          * Else send him to MainScreenActivity */
         (new Handler()).postDelayed(() -> {
             if (user != null) {
-                if (user.isEmailVerified() || user.isAnonymous()) {
+                // TODO: I can't test with this can we remove it for now while keeping the TODO ?
+                /*if (user.isEmailVerified() || user.isAnonymous()) {*/
                     startActivity(new Intent(MainActivity.this, MainScreenActivity.class));
-                } else {
+                /*} else {
                     startActivity(new Intent(MainActivity.this, LoginEmail.class));
-                }
+                }*/
```

Comment:
I think its better to keep this one uncommented and to not test for now as commenting this makes our app less usable and more buggy potentially

Reply:
I don't understand when this is un commentés i Just can't use thé app. Once i start it it takes me to the loginEmail class but si ce i log in with Google I can't login with e-mail and password. What I removed was Just the part that check if thé email is vérifié which we can't do right now

Reply:
does it still happen after wiping out the data of the emulator ? We need to make the back button go to the login activity and not the main activity in that case though thats something to do for another time.
If the issue is still there after wiping the data tell me what to do to reproduce the issue and i can try to find a fix but im not sure we can let every non email verified or non anonymous user get on the mainscreen safely

Reply:
It is not for testing on the emulator but with live Firebase
Like when i start the app myself it takes me to the loginEmail which i can't use.
To avoid that I have to wipe the app data everytime but this is costly when I just want to see if one button works.
I personnally believe that we should either find a way to verify our emails or disable this while the app is not available to gain a ton of time during manual testing. 
But I guess if I am the only one complaining it must be that I am missing something. How do you do manual/visual tests ? doesn't it redirect you to loginEmail ?